### DISCUSSION TITLE
Unable to start image Build 220302-0059f429

### DISCUSSION THREAD

**Discussion by juxeii** [photoprism/photoprism#2131] [Q&A] [answered]
I am using photoprism with portainer and today I wanted to pull latest image Build 220302-0059f429.
After recreating with the latest image, the container won't start.
Logs give:

```
init as root
image: amd64-prod, build 220302-bullseye
home: /photoprism
/entrypoint.sh: line 104: audit.sh: command not found
umask: "0002" (u=rwx,g=rwx,o=rx)
install-path: /opt/photoprism
running as uid 0
photoprism start
```
Any ideas what went wrong? All previous images were fine.

**EDIT**: With image photoprism:20220121 everything is fine, and logs are

```
started as root
running as root
photoprism start
time="2022-03-08T18:42:06Z" level=info msg="webdav: /originals/ enabled, waiting for requests"
time="2022-03-08T18:42:06Z" level=info msg="webdav: /import/ enabled, waiting for requests"
time="2022-03-08T18:42:06Z" level=info msg="http: starting web server at 0.0.0.0:2342"
```

**Comment by lastzero** [comment, marked as answer]
See #2120

**Reply by juxeii** [reply]
Thx, sorted it out. With portainer it is simply to restart the stack.
On Synology there is a bug where one cannot edit env varibles: [click](https://github.com/portainer/portainer/issues/2423)
Pretty annoying!
